pandas/io/parquet.py

```diff
@@ -512,17 +511,21 @@ def read_parquet(
     DataFrame
     """
     impl = get_engine(engine)
-
-    use_nullable_dtypes = (
-        use_nullable_dtypes
-        if use_nullable_dtypes is not lib.no_default
-        else using_nullable_dtypes()
-    )
+    if use_nullable_dtypes is not lib.no_default:
+        warnings.warn(
+            "The argument 'use_nullable_dtypes' is deprecated. Use "
+            "dtype_backend='numpy_nullable' instead of use_nullable_dtype=True.",
```

Comment:
I think the message should be structured like:

```
msg = "The argument 'use_nullable_dtypes' is deprecated and will be removed in a future version."
if use_nullable_dtypes is True:
   msg + = "Use dtype_backend='numpy_nullable' instead of ..."
```

Reply:
changed